quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java

```diff
@@ -339,6 +331,13 @@ protected void onPause() {
 
     @Override
     protected void onDestroy() {
+        /*
+         * Tear down audio management and restore previous volume stream
+         */
+        audioDeviceSelector.deactivate();
```

Comment:
You can get rid of this line if you are calling it right before stop as stop deactivates for you